autoload/SpaceVim/layers/test.vim

```diff
@@ -39,3 +39,13 @@ function! SpaceVim#layers#test#config() abort
   let g:test#custom_strategies = {'spacevim': function('SpaceVim#plugins#runner#open')}
   let g:test#strategy = 'spacevim'
 endfunction
+
+function! SpaceVim#layers#test#set_variable(var) abort
+  let l:override = get(a:var, 'override_config', {})
+  if !empty(l:override)
+    for l:option in keys(l:override)
+      let l:varname = 'test#'.substitute(l:option, '-', '#', 'g')
```

Comment:
here it should be `substitute(l:option, '_', '#', 'g')`

Reply:
Done. I also updated the docs to document the change accordingly.